appveyor.yml

```diff
@@ -5,15 +5,17 @@ environment:
 install:
   - ps: Install-Product node $env:nodejs_version x64
   - node --version
-  - yarn --version
-  - yarn install
+  - curl -fsSL -o yarn.js https://github.com/yarnpkg/yarn/releases/download/v0.27.5/yarn-0.27.5.js
```

Comment:
Then we would need to be sure that npm global dir precedes appveyor's global Yarn installation in PATH.
I don't have a strong opinion on this and happy to change to anything that maintainers think is best.